documentation/src/main/wiki/ProductionUsage.md

```diff
@@ -152,3 +149,30 @@ For instance, a configuration JSON for the *repository* verticle could look like
   }
 }
 ```
+#####Preserving headers passed to microservices
+Single service configuration allows to define which headers should be passed to microservices. 
```

Comment:
This configuration should say which headers we pass **from** microservices **to** end user via final request.